schemas/v1/task-group-defined-message.yml

```diff
@@ -0,0 +1,28 @@
+$schema:  http://json-schema.org/draft-06/schema#
+title:        "Task Group Defined - Create Status"
+description: |
+  Create a commit status on GitHub.
+  Full specification on [GitHub docs](https://developer.github.com/v3/repos/statuses/#create-a-status)
```

Comment:
This description is from the perspective of the message receiver, but as an API definition, the service is the *sender* and the user is the receiver.  That is, as someone reading this description I can't decide to send a message to create a commit status.  My only option is to decide to listen for messages like this.  So, what would it mean for me to receive a message like this?

I think it's important not to suggest that this is the task-group equivalent of `queue.taskDefined`, since someone might otherwise get the wrong idea and depend on it for that purpose.

Come to think of it, one we drop status support, we will likely want to get rid of this message.  So perhaps the description should just say something like "Indicates that this service has created a new task group in response to a GitHub event.  This message is for internal use only and should not be relied on for other purposes."